tests/unit/models/reddit/test_subreddit.py

```diff
@@ -102,6 +103,20 @@ def test_set__redundant_arguments(self):
         assert str(excinfo.value) == message
 
 
+class TestSubredditModmail(UnitTest):
```

Comment:
Just FYI, I usually consider atomic commits to include both the new code, and the tests for that new code. Splitting code from its tests is what one might call a "horizontal" split.

To think of it another way, if there were a reason to revert this one feature (assume the PR contains a few features), it's easier to revert a single commit than one commit for the feature, and another for its tests.